Makefile

```diff
@@ -405,6 +415,7 @@ docker-push-all: $(addprefix docker-push-,$(ALL_ARCH))
 	$(MAKE) docker-push-core-manifest
 	$(MAKE) docker-push-kubeadm-bootstrap-manifest
 	$(MAKE) docker-push-kubeadm-control-plane-manifest
+	$(MAKE) docker-push-operator-manifest
```

Comment:
I would argue that this task should be independent from core Makefile. Until the operator is experimental, default push command should accidentally create new images in release.